tests/test_tuning_XXXMBE.cpp

```diff
@@ -0,0 +1,157 @@
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include <string>
+#include <vector>
+#include <map>
+#include <limits>
+#include <algorithm>
+
+#define BOOST_TEST_MODULE TestTuningXXXMBE
+#include <boost/test/unit_test.hpp>
+
+struct Column : public std::vector<std::string> {
+		Column(const std::string& name_, const int size = 0, const int num_rows_estimate = 1000) : 
+				std::vector<std::string>(size), name(name_)
+		{
+				this->reserve(num_rows_estimate);
+		}
+
+		// Return vector of double values, invalid elements set to NaN
+		std::vector<double> dvalues() const {
+				std::vector<double> vec;
+				vec.reserve(this->size());
+				
+				const auto& conv_func = [](const std::string& strval) {
+						double dval;
+						try {
+								dval = std::stod(strval);
+						} catch (std::invalid_argument& exc) {
+								dval = std::numeric_limits<double>::quiet_NaN();
+						}
+						return dval;
+				};
+
+				std::transform(this->cbegin(), this->cend(), std::back_inserter(vec), conv_func);
+				return vec;
+		}
+
+		// Return vector of double values values, invalid elements set to std::numeric_limits<int>::min()
+		std::vector<int> ivalues() const {
+				std::vector<int> vec;
+				vec.reserve(this->size());
+				
+				const auto& conv_func = [](const std::string& strval) {
+						int ival;
+						try {
+								ival = static_cast<int>(std::stod(strval));
```

Comment:
Is there a reason for not using [`std::stoi()`](https://en.cppreference.com/w/cpp/string/basic_string/stol) here?

Reply:
except ignorance, no.. 😳